client/hosting/server-settings/components/sftp-card/index.tsx

```diff
@@ -133,9 +126,19 @@ export const SftpCard = ( {
 				requestSshAccess( siteId );
 			}
 		}
-		return onDestroy();
```

Comment:
This does indeed look like a mistake, even if it's hard to tell for sure at first glance. Did you verify this somehow, @nightnei?

Reply:
Yes, as a result - we didn't clean it up during unmounting and it was possible to notice it for a short period of time when we did the second mounting (go back and forth between pages). Why for the short period of time - since we retrieve the list of users during mounting, o it wasn't a big issue. But for safety reason - it' better to clean it up as soon as possible (it was what the author intended to do, but called onDestroy by mistake, and called it with wrong arguments - userId instead of username).